hera_cal/tests/mock_uvdata.py

```diff
@@ -0,0 +1,284 @@
+"""Functions for mocking UVdata objects for testing purposes."""
+from __future__ import annotations
+
+from pyuvdata import UVData, UVCal
+from pyuvdata.uvdata import FastUVH5Meta
+from hera_cal import utils
+import numpy as np
+from astropy.coordinates import EarthLocation
+import yaml
+from hera_cal.data import DATA_PATH
+from pathlib import Path
+from hera_cal.red_groups import RedundantGroups
+
+HERA_LOC = EarthLocation.from_geodetic(
+    lat=-30.721526120690307,
+    lon=21.428303826863015,
+    height=1051.690000070259,
+)
+
+with open(f"{DATA_PATH}/hera_antpos.yaml", "r") as fl:
+    HERA_ANTPOS = yaml.safe_load(fl)
+
+
+def create_mock_hera_obs(
+    jdint: int = 2459855, 
+    integration_time=10.7,
```

Comment:
I'd suggest making the default parameters exactly match real data... for example, this integration time is wrong

Reply:
Good point. I've changed this to 9.66xxx and the frequencies to match the phase II array.